pkg/values/validation/validator_test.go

```diff
@@ -0,0 +1,60 @@
+package validation
+
+import (
+	"github.com/flant/addon-operator/pkg/utils"
+	"testing"
+
+	. "github.com/onsi/gomega"
+)
+
+//func prepareConfigObj(g *WithT, input string) *VersionedUntyped {
```

Comment:
It will be better to delete the commented code.

Reply:
Yep